examples/decomposition/plot_faces_decomposition.py

```diff
@@ -34,8 +34,9 @@
 
 # #############################################################################
 # Load faces data
-dataset = fetch_olivetti_faces(shuffle=True, random_state=rng)
-faces = dataset.data
+faces, _ = fetch_olivetti_faces(
+    return_X_y=True, shuffle=True, random_state=rng
+)
```

Comment:
Nit

```py
faces, _ = fetch_olivetti_faces(return_X_y=True, shuffle=True, 
                                random_state=rng)
```

Reply:
I'm sure the insidious inclusion of black code style into the code base to get reviewers used to it over time won't be foiled so easily :)

Reply:
Ah I see what is going on here.

Reply:
:) but both way work